core-aam/core-aam.html

```diff
@@ -1816,7 +1843,7 @@
 		   <p>IAccessible2: </p>
 		   <p>If the referenced object is in the accessibility tree, expose a pointer to the accessible object using <code>IA2_RELATION_DETAILS</code>, and expose reverse relations as described in <a href="#mapping_additional_relations">Relations</a>.</p>
 		  </td>
-                  <td>TBD.</td>
+                  <td><code>DescribedBy</code> property points to the referenced element</td>
```

Comment:
In light of the [previous discussion](https://github.com/w3c/aria/pull/469#issuecomment-260018111), how about changing this to:
"`DescribedBy` array property contains an entry pointing to the referenced accessible object".